tests/unit/test_imagenet_format.py

```diff
@@ -212,37 +211,40 @@ def _create_expected_dataset(self):
         )
 
     @mark_requirement(Requirements.DATUM_GENERAL_REQ)
-    def test_can_import(self):
+    @pytest.mark.parametrize("dataset_cls, is_stream", [(Dataset, False), (StreamDataset, True)])
+    def test_can_import(self, dataset_cls, is_stream, helper_tc):
         expected_dataset = self._create_expected_dataset()
-        dataset = Dataset.import_from(self.DUMMY_DATASET_DIR, self.FORMAT_NAME)
+        dataset = dataset_cls.import_from(self.DUMMY_DATASET_DIR, self.IMPORTER_NAME)
+        assert dataset.is_stream == is_stream
 
-        compare_datasets(self, expected_dataset, dataset, require_media=True)
+        compare_datasets(helper_tc, expected_dataset, dataset, require_media=True)
 
     @mark_requirement(Requirements.DATUM_GENERAL_REQ)
     def test_can_detect_imagenet(self):
         detected_formats = Environment().detect_dataset(self.DUMMY_DATASET_DIR)
-        self.assertEqual([self.IMPORTER_NAME], detected_formats)
+        assert [self.IMPORTER_NAME] == detected_formats
```

Comment:
Overall changes look good to me. Is there any reason to use `assert` directly instead of using `self.asserEqual` of `unittest`?

Reply:
The class is not a subclass of TestCase anymore, so it does not have `assertEqual` method